configuration/identities/identityGroups.sqf

```diff
@@ -121,6 +120,43 @@ _names = [_forenames, _surnames];
 CREATE_IDENTITY(latino,_speakers,_faces,_names);
 
 
+// Add GM identities
+if !(isNil 'gm_core_characters_fnc_uc_getRankTexture') then
```

Comment:
Function names may change. Consider checking for GM's mod class instead (should be safer):

```sqf
isClass (configfile >> "CfgMods" >> "GM")
```

Reply:
I am much in favor of the written warning, as to not overload the config files with code